single-function/add-two-numbers/02-impl-revision.js

```diff
@@ -1,24 +1,24 @@
 const addTwoNumbers = (l1, l2) => {
```

Comment:
`l1` is a little hard to read since 'l' and '1' are pretty similar.

I think we should encourage folks to write out names in general for readability. A few keystrokes in this case is worth the readability.  We could even have a list of accepted exceptions: `i`, `j`, `idx`, `val`, etc. 

So in this case maybe `list1` and `list2`?  or `listA` and `listB`?

Reply:
I like listA